bpf/lib/nat.h

```diff
@@ -738,10 +738,18 @@ snat_v4_needs_masquerade(struct __ctx_buff *ctx __maybe_unused,
 	 */
 	remote_ep = lookup_ip4_remote_endpoint(tuple->daddr, 0);
 	if (remote_ep && identity_is_remote_node(remote_ep->sec_identity)) {
+		/* SNAT the packet to remote node if enable-remote-node-masquerade flag is set to true.
+		 * This is a feature flag that can be enabled in BPF masquerading mode.
+		 */
+		#ifdef ENABLE_REMOTE_NODE_MASQUERADE
```

Comment:
could you please also wire up some build testing (in `bpf/Makefile`) and complexity testing (in `bpf/complexity-tests/.../bpf_host/...`) for this symbol?

Reply:
Added both requested tests in the new commit